lib/fastlane/plugin/mango/helper/mango_helper.rb

```diff
@@ -155,8 +156,12 @@ def get_container_instance(container)
       def create_container_call
         # When CPU is under load we cannot create a healthy container
         wait_cpu_to_idle
-
-        emulator_args = is_running_on_emulator ? "-p #{no_vnc_port}:6080 -e DEVICE='#{device_name}'" : ''
+        additionalenv = ''
+        environment_variables.each do |variable|
+          additionalenv = additionalenv + " -e #{variable}"
+        end
+        emulator_args = is_running_on_emulator ? "-p #{no_vnc_port}:6080 -e DEVICE='#{device_name}'#{additionalenv}" : ''
```

Comment:
this way it will add env vars only in case we're running on emulator. you can modify emulator args after this line is executed, to have vars there for all the cases